packages/plugins/plugin-navtree/src/components/L0Menu.tsx

```diff
@@ -191,37 +207,37 @@ const L0Item = ({ item, parent, path, pinned, userAccount, onRearrange }: L0Item
           hue={item.properties.hue}
           emoji={item.properties.emoji}
           status={item.properties.status}
+          size={10}
         />
       ) : (
         <div
           role='none'
           data-frame={true}
           className={mx(
-            'absolute grid dx-focus-ring-group-indicator transition-colors',
-            type === 'tab' || pinned ? 'rounded' : 'rounded-full',
+            'absolute grid dx-focus-ring-group-indicator transition-colors rounded',
             pinned
-              ? 'bg-transparent group-hover/l0i:bg-groupSurface inset-inline-3 inset-block-0.5'
-              : 'bg-groupSurface inset-inline-3 inset-block-2',
+              ? 'bg-transparent group-hover/l0item:bg-groupSurface inset-inline-1 inset-block-0.5'
```

Comment:
What is motivating changes like this, increasing the size of the L0 avatars and reducing negative space? Does this not feel too tight with the accent notch? Is this just a matter of personal preference or is there a precedent or design principle that a change like this intends to carry out?